src/char_mapping.rs

```diff
@@ -94,6 +164,50 @@ table! {
     }
 }
 
+table! {
+    #[doc = "A char-to-glyph encoding in format 12."]
+    pub Encoding12 {
+        format      (u16), // format
+        reserved    (u16) = { 0 }, // reserved
+        length      (u32), // length
+        language    (u32), // language
+        num_groups  (u32), // numGroups
+
+        groups (Vec<SequentialMapGroup>) |this, tape| { // groups
+            tape.take_given(this.num_groups as usize)
+        },
+    }
+}
+
+table! {
+    #[doc = "A char-to-glyph encoding in format 14."]
+    pub Encoding14 {
+        format               (u16), // format
+        length               (u32), // length
+        num_var_selectors    (u32), // numVarSelectorRecords
```

Comment:
Again, there are extra spaces here.

Reply:
Let’s rename `num_var_selectors` to `variation_selector_count` (see my other comments about naming conventions).